src/main/java/org/libsdl/app/SDLActivity.kt

```diff
@@ -237,7 +237,7 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
     @Suppress("unused")
     fun inputGetInputDeviceIds(sources: Int): IntArray {
         return InputDevice.getDeviceIds().fold(intArrayOf(0)) { result: IntArray, id: Int ->
-            val device = InputDevice.getDevice(id)
+            val device = InputDevice.getDevice(id) ?: return result
```

Comment:
the weird thing is that this is not causing an issue when building the sample app or the standalone player. I only run into this when building the app. Kotlinversion and sdks were all aligned, still no compilation error. The new variant is also building from the sample app and the player though.